### ISSUE TITLE
What sort of loss curves should we expect?

### ISSUE THREAD

**Issue by cinjon** [wzmsltw/BSN-boundary-sensitive-network.pytorch#17]
When training, at approximately what level of loss for each of the parts does the model start to become reasonable? And what kind of curves should we expect?

**Comment by wzmsltw** [comment]
It actually depends on dataset...  In small dataset such as THUMOS-14, the overfitting is happening soon (about 3-4 epochs). You should decay the learning rate by 0.1 and take the first epoch model after decaying.

**Comment by cinjon** [comment]
Any chance you could show a few common loss curves? Most helpful would be the ones for Thumos using your representations from UCF101.
